target/linux/generic/files/drivers/net/phy/rtl8367b.c

```diff
@@ -1580,15 +1633,36 @@ static struct rtl8366_smi_ops rtl8367b_smi_ops = {
 	.enable_port	= rtl8367b_enable_port,
 };
 
+#ifdef CONFIG_OF
+static const struct of_device_id rtl8367b_match[] = {
+	{ .compatible = "realtek,rtl8367b" },
+	{
+		.compatible = "realtek,rtl8367c",
+		.data = (void *)RTL8367B_FLAG_MATCH_CHIP_NUM,
```

Comment:
This is not how .data should be used. Point to a const struct and not abuse the pointer as an int.